db/migrate/20230111102451_add_teaching_authority_provides_written_statement.rb

```diff
@@ -0,0 +1,11 @@
+class AddTeachingAuthorityProvidesWrittenStatement < ActiveRecord::Migration[
```

Comment:
```suggestion
class AddTeachingAuthorityProvidesWrittenStatementToRegions < ActiveRecord::Migration[
```

The convention is to include the table name in the migration name.